### DISCUSSION TITLE
SessionGuard remember_me cookie name

### DISCUSSION THREAD

**Discussion by dragonofmercy** [laravel/framework#50187] [Ideas]
Is it possible to change the function getRecallerName() where the cookie name is hard coded to a config based option

Original:
https://github.com/laravel/framework/blob/08bf276de0efc9af130e10cb93ede827715543c3/src/Illuminate/Auth/SessionGuard.php#L807-L810

New version (example):

```php
public function getRecallerName()
{
    return config('auth.drivers.session.remember_cookie');
}
```

**Comment by henzeb** [comment]
what would be the reason to change this? As a work around, you could extend the SessionGard with your own `getRecallerName` and then extend the AuthManager with it...

**Reply by dragonofmercy** [reply]
It's just to be consistent, session cookie can be renamed with SESSION_COOKIE env variable, why not this one.  
I have done the workaround to extend authmanger but I think it's a lot of code just to rename a cookie.
